print-dhcp6.c

```diff
@@ -784,6 +790,43 @@ dhcp6opt_print(netdissect_options *ndo,
 			ND_PRINT(")");
 			break;
 
+		case DH6OPT_BOOTFILE_URL:
+			if (optlen < 2) {
+				ND_PRINT(" ?)");
+				break;
+			}
+			tp = (const u_char *)(dh6o + 1);
+			ND_PRINT(" ");
+			nd_printjnp(ndo, tp, optlen);
+			ND_PRINT(")");
+			break;
+
+		case DH6OPT_SZTP_REDIRECT:
+		case DH6OPT_USER_CLASS:
+			ND_PRINT(" ");
+			tp = (const u_char *)(dh6o + 1);
+			first_list_value = TRUE;
+			remainder_len = optlen;
+			while (tp < cp + sizeof(*dh6o) + optlen - 2) {
+				if (first_list_value == FALSE) {
+					ND_PRINT(",");
+				}
+				first_list_value = FALSE;
+				subopt_len = GET_BE_U_2(tp);
+				if (subopt_len > remainder_len-2) {
```

Comment:
This would be a good place to print another "invalid" marker so the user can see something is wrong with the packet.

Reply:
This falls through to the "remainder_len != 0" if below, which results in "?)", the common truncation marker in this file.